app/com/gu/floodgate/AppComponents.scala

```diff
@@ -78,7 +78,6 @@ class AppComponents(context: Context)
   val reindexService = new ReindexService(
     contentSourceService,
     runningJobService,
-    jobHistoryService,
```

Comment:
This isn't used anymore? It's been a while since I was looking around in this code but I thought this was where the history of the tasks was recorded.

Reply:
> This isn't used anymore? It's been a while since I was looking around in this code but I thought this was where the history of the tasks was recorded.

jobHistoryService seems to be used in ProgressTracker.
The history table does still seem to record new jobs.

This PR is just cleaning the already unused val so I think it's safe.